### DISCUSSION TITLE
是否能够支持对group的编辑呢

### DISCUSSION THREAD

**Discussion by huali-dong** [PengChen96/ajax-tools#12] [Ideas]
比如对group整体设置request header

**Comment by PengChen96** [comment]
> 比如对group整体设置request header

你是想一整个group都设置相同的header这样吗

**Reply by huali-dong** [reply]
对 我想一整个group都设置相同的header

**Reply by huali-dong** [reply]
还有一个问题也想问： 这个能设置response status吗？ 比如我想模拟后端接口返回404这种

**Comment by PengChen96** [comment]
> 比如对group整体设置request header

比如这个group设置header，匹配接口时使用正则去匹配接口，这样去设置header能解决你的这个需求吗

**Reply by huali-dong** [reply]
可以的

**Comment by PengChen96** [comment]
> 还有一个问题也想问： 这个能设置response status吗？ 比如我想模拟后端接口返回404这种
这个暂时是不支持

**Reply by huali-dong** [reply]
那如果后端接口返回 401没权限这种，mock也改变不了了吗？

**Reply by PengChen96** [reply]
现在我是只能设置status=200，需要修改status要改下代码

**Reply by huali-dong** [reply]
哈哈 那麻烦作者 加一个能改response status的入口哇， 这样用的更多一点 😂

**Reply by PengChen96** [reply]
可以，我后面改改

**Reply by PengChen96** [reply]
这个我刚更新了，你在这下载安装试试 [https://github.com/PengChen96/ajax-tools/blob/master/kphegobalneikdjnboeiheiklpbbhncm.crx](https://github.com/PengChen96/ajax-tools/blob/master/kphegobalneikdjnboeiheiklpbbhncm.crx)
<img width="498" alt="image" src="https://github.com/PengChen96/ajax-tools/assets/16712630/146908da-7632-4be4-8b61-61f160e31622">

**Reply by huali-dong** [reply]
感谢作者 真速度 👍

**Reply by PengChen96** [reply]
😁
